search/src/main/kotlin/com/alfresco/content/search/components/ComponentCreateViewModel.kt

```diff
@@ -42,6 +44,17 @@ class ComponentCreateViewModel(
         } else updateSingleComponentData("", "")
     }
 
+    /**
+     * update the value for number range
+     */
+    fun updateFormatDateRange() = withState {
+        if ((fromDate.isNotEmpty() && toDate.isNotEmpty())) {
+            val dateFormat = context.getLocalizedName("$fromDate - $toDate")
```

Comment:
Please check this statement, is it required?

Reply:
Done